packages/@o3r/localization/builders/helpers/localization.generator.ts

```diff
@@ -123,14 +124,18 @@ export class LocalizationExtractor {
       })
       .map(({ref}) => {
         if (!ref.startsWith('.')) {
-          try {
-            return require.resolve(ref);
-          } catch {
-            // Should be a local file
+          if (this.options?.libraries?.length && this.options.libraries.every((lib) => !ref.startsWith(lib))) {
```

Comment:
shouldn't it be:
```suggestion
          if (!this.options?.libraries?.length || this.options.libraries.every((lib) => !ref.startsWith(lib))) {
```
or am I misunderstanding ?
it seems we use require only if the ref is not in the list of options.libraries, so I would expect to do it also if the list is empty

Reply:
When there are no libraries we don't want to `require.resolve`, and we don't want too if we know the library because we will already get the information from its metadata